From 5a88b98511047202ce55d4089558a8ef1709d1a5 Mon Sep 17 00:00:00 2001
From: Jelle van der Waa <jelle@archlinux.org>
Date: Fri, 14 Apr 2023 19:05:56 +0200
Subject: [PATCH] test: fix redirect to https

curl does not redirect automatically making the test suite fail to
download the tar.gz.
---
 test/svg/Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/test/svg/Makefile b/test/svg/Makefile
index 7ac9b685..d80f6e11 100644
--- a/test/svg/Makefile
+++ b/test/svg/Makefile
@@ -10,4 +10,4 @@ clean:
 
 suite:
 	mkdir suite
-	(cd suite; curl -O http://www.w3.org/Graphics/SVG/Test/20110816/archives/W3C_SVG_11_TestSuite.tar.gz; tar xzvf W3C_SVG_11_TestSuite.tar.gz)
+	(cd suite; curl -O https://www.w3.org/Graphics/SVG/Test/20110816/archives/W3C_SVG_11_TestSuite.tar.gz; tar xzvf W3C_SVG_11_TestSuite.tar.gz)
-- 
2.40.0

